WordPress/WordPressUITests/Tests/EditorAztecTests.swift

```diff
@@ -1,3 +1,4 @@
+import UITestsFoundation
```

Comment:
This is the kind of `import` I mentioned in my comment above. There's only a handful of changes here compared to the quantity of tests that we have because most tests already import the framework because of other shared functionalities that were there already.